<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>network.png</normaloff>network.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>281</width>
      <height>691</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Initial Condition</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>layer A</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>38</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Node</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>64</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Internal Edge</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>92</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>External Edge</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>121</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Structure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>148</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>State</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>178</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>MAX</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>65</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="maxVisibleItems">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>93</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>147</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>[1, 2]</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>177</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>+2</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>38</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>2048</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_15">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>-2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>MIN</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>251</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>layer B</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Node</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>58</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Internal Edge</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>85</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>External Edge</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>112</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Structure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>141</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>State</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>84</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_8">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>112</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>58</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_10">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>142</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>[-1]</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_11">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>31</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="maxVisibleItems">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>2048</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>251</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Simulation Condition</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>32</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Step</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>67</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Drawing</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Database</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_12">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>35</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_13">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>False</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>True</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_14">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>102</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>renew_competition</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>competition</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Structures</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_16">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>RR-RR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BA-BA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RR-BA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BA-RA</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="condition_settingButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>650</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="Total">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>781</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Initial State</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>761</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>520</y>
        <width>341</width>
        <height>32</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Result</string>
     </attribute>
     <widget class="QDialogButtonBox" name="buttonBox_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>530</y>
        <width>341</width>
        <height>32</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>쪽</string>
     </attribute>
     <widget class="QDialogButtonBox" name="buttonBox_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>510</y>
        <width>341</width>
        <height>32</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>쪽</string>
     </attribute>
     <widget class="QDialogButtonBox" name="buttonBox_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>520</y>
        <width>341</width>
        <height>32</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>쪽</string>
     </attribute>
     <widget class="QDialogButtonBox" name="buttonBox_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>520</y>
        <width>341</width>
        <height>32</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
